packages/client/src/v2-events/features/events/registered-fields/LocationSearch.tsx

```diff
@@ -167,7 +168,7 @@ function LocationSearchOutput({ value }: { value: Stringifiable }) {
     .filter(Boolean)
     .reverse()
 
-  return joinValues([name, ...resolvedAdminLevels, country], ', ')
+  return joinValues(uniq([name, ...resolvedAdminLevels, country]), ', ')
```

Comment:
I think this might work a bit inconsistently in cases where, for example, the name of a city is the same as the state it’s in. I don’t think this is too uncommon.

In those cases we would probably still want to show both the leaf location and the district/state.

Real life example: New York, New York

I'm going to approve anyway, up to you if you want to do something about it or not 👍